test/python/utils/db_client_psql.py

```diff
@@ -0,0 +1,48 @@
+import psycopg2
+
+#format strings for logging
+line_formatter = "*" * 30 + "{}" + "*" * 30
+connect_server_formatter = " Connected to {} Server successfully!! "
+close_connection_formatter = " Connection closed to {} Server successfully!! "
+
+# make connection object using psycopg
+class Db_Client_psycopg:
```

Comment:
Writing psql connection as part of existing python tests led to installing psycopg in python tests workflow and tsql dependencies in this workflow despite of not being used. So to avoid that I created a separate file.

Reply:
I think having ability to open PSQL connection is a good add on to python framework but can be done later as a refactoring